assets/dev/js/editor/controls/gallery.js

```diff
@@ -104,9 +109,9 @@ ControlMediaItemView = ControlBaseDataView.extend( {
 
 	imageNotOptimized( attachment ) {
 		const checks = {
-			height: 1200,
-			width: 1200,
-			filesizeInBytes: 200000,
+			height: 1080,
+			width: 1920,
+			filesizeInBytes: 100000,
 		};
```

Comment:
Have you considered adding a filter hook to this area?

Reply:
nope, its fixed by design